### DISCUSSION TITLE
Separate binary for beamer editing.

### DISCUSSION THREAD

**Discussion by jingkaimori** [MoganLab/mogan#426] [Ideas]
Audio and video are needed in presentations, but a typical document does not need multimedia support. Is it possible to provide a separate binary optimized for presentation built with multimedia support?

**Comment by YiqiXu** [comment]
Audios and videos are indispensable for websites. They should also be considered for this situation since TeXmacs can export HTML files.

**Comment by da-liii** [comment]
We can discuss here: http://forum.texmacs.cn/c/derivatives
